### ISSUE TITLE
How long it takes in a non GPU system

### ISSUE THREAD

**Issue by nkathireshan** [rohithreddy024/Text-Summarizer-Pytorch#27]
Hi Rohit,

Thanks for the repo, I have a non-GPU machine with 32 GB RAM, Windows 10 OS, I ran the train.py and it is running for more than 12 hours now and just the first iteration is running.
i.e iter: 1000 mle_loss: 4.652 reward: 0.0000

Just wondering if the code is running correctly? and also wanted to ask, how long it will take for a complete execution?
Is GPU mandatory to run the training ?

**Comment by nkathireshan** [comment]
![image](https://user-images.githubusercontent.com/26978746/79766184-431a4a80-8345-11ea-918b-30de6f08335d.png)

Its been more than 4 hours, and its still the same ? do I have to pass any input here ? this looks like it needs an input ?

**Comment by pengzhi123** [comment]
if u want to get the train data, u need have python 2.7 and tensorflow. and use this codes, should have python3 , ubuntu , Pytorch 0.4.1 and cuda 9.0.

**Comment by pengzhi123** [comment]
No GPU will cause training to take more than 20 days, I used 16gb 1080ti gpu, spent 2 and a half days training, and performed 500,000 iterations. About 100,000 times the model has converged. If you have a GPU, you should install cuda9.0.

**Comment by nkathireshan** [comment]
> No GPU will cause training to take more than 20 days, I used 16gb 1080ti gpu, spent 2 and a half days training, and performed 500,000 iterations. About 100,000 times the model has converged. If you have a GPU, you should install cuda9.0.

Wow Great, thank you for your response.
